tests/action_test_params.py

```diff
@@ -1318,6 +1318,33 @@
 
 """)
 
+STORE_RETRIEVE_TO_OBJECT_ALREADY_SAMEVALUE = yaml.safe_load("""
+actions:
+- Create-object:
+    Type: Enterprise
+    Actions:
+    - Set-values:
+        name: enterprise1
+    - Create-object:
+        Type: Infrastructure Access Profile
+        Actions:
+            - Set-values:
+                infrastructure_access_profile_name: access1
+                ssh_auth_mode: password_and_key_based
+                user_name: admin
+                password: abcd1234
+                source_ip_filters: ["1.1.1.1", "2.2.2.2", "3.3.3.3"]
+                ssh_key_names: ["key1"]
+                ssh_keys: ["japudofiuasdfoiudpfou"]
+    - Create-object:
```

Comment:
This needs to be a "select-object".  As it stands, you are creating two different objects and there is no conflict.